SerialPrograms/Source/PokemonSV/Programs/ItemPrinter/PokemonSV_ItemPrinterRNG.cpp

```diff
@@ -774,6 +811,48 @@ void ItemPrinterRNG::program(SingleSwitchProgramEnvironment& env, BotBaseContext
         stats.iterations++;
         env.update_stats();
     }
+}
+
+void ItemPrinterRNG::program(SingleSwitchProgramEnvironment& env, BotBaseContext& context){
+    assert_16_9_720p_min(env.logger(), env.console);
+    ItemPrinterRNG_Descriptor::Stats& stats = env.current_stats<ItemPrinterRNG_Descriptor::Stats>();
+    env.update_stats();
+
+
+
+    if (!AUTO_MATERIAL_FARMING){
+        run_item_printer_rng(env, context, stats);
+    }
+    else {

```

Comment:
nit: The coding convention we normally use is `}else{` and no spaces between `if (){`. Though we aren't super-consistent with adhering to it.